modules/alignment.nf

```diff
@@ -78,31 +77,39 @@ process alignReads {
     }
 
     input:
-        tuple val(email),
-              val(library),
-              path(input_file1),
-              path(input_file2),
-              val(fileType)
+        tuple val(library),
+        path(input_file1),
+        path(input_file2),
+        val(fileType)
         path(genome)
 
     output:
-        tuple val(params.email), val(library), env(barcodes), path("*.nonconverted.tsv"), path("*.fastp.json"), emit: for_agg
-        path "*.aln.bam", emit: aligned_bams
+        tuple val(library), path("*.fastp.json"), emit: fastp_reports
         tuple val(library), path("*.nonconverted.tsv"), emit: nonconverted_counts
-        tuple val(library), path("*.aln.bam"), path("*.aln.bam.bai"), env(barcodes), emit: bam_files
+        tuple val(library), path("*.aln.bam"), path("*.aln.bam.bai"), emit: aligned_bams
+        tuple val(library), env(barcodes), env(flowcell), env(num_reads_used), emit: metadata
```

Comment:
We DO still pass this channel to aggregation (also to multiqc). 
@lnblum @bwlang This is related to my comment in the aggregation module. If we are confident that the aggregation is not going to fail on extracting the barcode (and relating it to the right orientation based on the sequencer used) from metadata, I can delete this (and related lines in alignment module - e.g. barcode variable does not have to be exported to env at all).